doc/GUIDE.md

```diff
@@ -919,8 +619,8 @@ under the `extra-deps` section.
 
 #### Installing the compiler
 
-You can install the required compiler if not already installed by using the
-`--install-ghc` flag with the `stack init` command.
+stack will automatically install the compiler when you run `stack build` but you can
+manually specify the compiler by running `stack setup <GHC-VERSION>`.
```

Comment:
Think this needs to be here, the manually specifying the GHC version part? At least if it stays it seems like it should be a separate sentence to me, and maybe mention that if they're trying to change what GHC is used by a package, they should modify the resolver in stack.yaml instead of hoping that'll do it.